Another-application/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# escape=`
+FROM microsoft/iis
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'Continue'; $verbosePreference='Continue';"]
+
+
+RUN install-windowsfeature Web-Asp-Net45
+
+RUN Install-WindowsFeature Web-Windows-Auth 
+
+RUN Restart-Service W3SVC 
+
+
```

Comment:
nit: extra lines in this file. Can you make it consistent with the other files in this repo